src/usr/local/www/services_ntpd.php

```diff
@@ -43,14 +43,7 @@
 }
 
 if (empty($config['ntpd']['interface'])) {
-	if (is_array($config['installedpackages']['openntpd']) && is_array($config['installedpackages']['openntpd']['config']) &&
```

Comment:
Can this be deleted?
Seems like legacy code, not actual for 2.5

Reply:
There are a few other references around to the old `openntpd` code around in various places. It has probably been gone long enough to remove at this point. Or we could setup some upgrade code to handle this upgrade rather than doing it here or anywhere else.

Either way though I'd prefer to see that in a separate issue+PR rather than combined with this change.